### PR TITLE
DB integration

### PR DESCRIPTION
**Pull request by aurangzaib048**
Resolves: https://github.com/brave/news-aggregator/issues/762

### CODE REVIEW

**Review by porteron — DISMISSED** [review]
Overall it looks clean. I've never used alembic before but it looks pretty straightforward. The migrations setup looks like it will develop nicely over time. I don't see any queries or migrations to create the database user(s). Maybe we should include that.

**Review by porteron — DISMISSED** [review]
Looks good

**Review by porteron — APPROVED** [review]
Looks good. Not sure if it exists somewhere already but it would be great to add to the README some generic information about the schema design, how and why we are using alembic, how migrations are managed.

File: db/README.md
Comment (porteron) [inline db/README.md:1]: Thank you 🙏

**Review by porteron — APPROVED** [review]

File: db/README.md
Comment (porteron) [reply db/README.md:1]: Super helpful

**Comment by aurangzaib048** [comment]
Infra PR: https://github.com/brave-intl/today-ops/pull/154

**Pull request merged**